airbyte-ci/connectors/connector_ops/tests/conftest.py

```diff
@@ -53,3 +54,8 @@ def dummy_qa_report() -> pd.DataFrame:
             }
         ]
     )
+
+
+@pytest.fixture()
+def set_working_dir_to_repo_root(monkeypatch):
```

Comment:
@alafanechere I discovered we are not currently running these tests.

I imagine its because many of them depend on connectors being preset in the main repo.

This was a shortcut I took to have them passing again. But I want to raise a flag that it is a hack. Just not sure if its an acceptable short term hack.

Thoughts?